lib/screens/edit_product_screen.dart

```diff
@@ -40,11 +40,23 @@ class _EditProductScreenState extends State<EditProductScreen> {
 
   void _updateImageUrl() {
     if (!_imageUrlFocusNode.hasFocus) {
+      if ((!_imageUrlController.text.startsWith('http') && !_imageUrlController.text.startsWith('https')) ||
+          (!_imageUrlController.text.endsWith('.png') &&
+              !_imageUrlController.text.endsWith('jpg') &&
+              !_imageUrlController.text.endsWith('.jpeg'))) {
+        return;
+      }
       setState(() {});
```

Comment:
remove this

Reply:
The setState cannot be removed. If the setState is removed, the preview of imageUrl wont be loaded unless the save button or the check button in the keyboard is pressed. 
setState is used to load the image when the user clicks other TextFormField aside from the imageUrl without the need to press the save button or check button.

![image](https://user-images.githubusercontent.com/74462760/104464916-6df0b100-55ee-11eb-849d-58357a61c021.png)